collectors/pod.go

```diff
@@ -137,6 +137,18 @@ var (
 		"The limit on memory to be used by a container in bytes.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerResourceRequestsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_requests_gpu",
+		"The number of requested gpu by a container.",
+		[]string{"namespace", "pod", "container", "node"}, nil,
+	)
+
+	descPodContainerResourceLimitsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_limits_gpu",
+		"The limit on gpu to be used by a container.",
```

Comment:
ditto.

`The limit on gpu to be used by a container.` -> `The limit on gpu cores to be used by a container.`